arraycontext/impl/pytato/compile.py

```diff
@@ -154,7 +154,34 @@ def id_collector(keys, ary):
     return pmap(arg_id_to_arg), pmap(arg_id_to_descr)
 
 
-def _get_f_placeholder_args(arg, kw, arg_id_to_name):
+def _transform_array(ary, actx):
+    """
+    Returns the transformed version of *ary* after calling
+    :meth:`ArrayContext.transform_dag` on it.
+    """
```

Comment:
The docstring doesn't totally reflect all the types that this function accepts, or why it's useful.

Reply:
Added more details to the docs. Thanks!